packages/miniflare/scripts/build.mjs

```diff
@@ -102,7 +102,7 @@ const embedWorkersPlugin = {
 					bundle: true,
 					sourcemap: true,
 					sourcesContent: false,
-					external: ["miniflare:shared", "miniflare:zod"],
+					external: ["miniflare:shared", "miniflare:zod", "cloudflare:workers"],
```

Comment:
why is `cloudflare:workers` marked as an external dependency here?

Reply:
because `cloudflare:workers` is a runtime thing, and therefore esbuild cannot resolve it at build time. Marking this as external is the correct way to go